layouts/partials/embedshare.haml

```diff
@@ -0,0 +1,45 @@
+%h3 Embed into your Website
+%input{:type => "text", :size=> "85", :readonly => true, 
```

Comment:
I'd suggest adding an on-focus-select-all handler to the text-field, like this:

    <iframe id="txtShareFrame" ...></iframe>
    
    $("#txtShareFrame").on('focus', function() {
        $(this).select();
    });